rpc/block_create.c

```diff
@@ -415,7 +415,9 @@ glusterBlockCreateRemoteAsync(blockServerDefPtr list, size_t mpath,
 
  out:
   for (i = 0; i < mpath; i++) {
-    GB_FREE(args[i].reply);
+    if (args[i]) {
```

Comment:
For this, it should be:

if (args && args[i]) {

Or just before the 'for (i = 0; i < mpath; i++) {' to check the args ?

Reply:
Please ignore this, it is the stale comment.